smart-contracts/application-typescript/src/serverProducer.ts

```diff
@@ -69,5 +98,8 @@ export class ServerProducer {
         app.delete('/certificate', async (req, res) => {
             res.sendStatus(403);
         });
+        app.delete('/farmer', async (req, res) => {
+            res.sendStatus(403);
```

Comment:
I guess that indeed a producer may not delete a farmer from the ledger. However, a small line of explanation might clarify this decision a little better